server/notification-providers/opsgenie.js

```diff
@@ -43,8 +43,8 @@ class Opsgenie extends NotificationProvider {
 
             if (heartbeatJSON.status === DOWN) {
                 let data = {
-                    "message": monitorJSON ? textMsg + `: ${monitorJSON.name}` : textMsg,
-                    "alias": monitorJSON.name,
+                    "message": monitorJSON ? textMsg + `: ${monitorJSON.pathName}` : textMsg,
+                    "alias": monitorJSON.pathName,
```

Comment:
no idea if special characters are allowed here

```suggestion
                    "alias": monitorJSON.name,
```